src/tests/server.ts

```diff
@@ -1,26 +1,32 @@
-import { Response, Router } from "express"
+import { Response, Router, urlencoded } from "express"
 import multer from "multer"
 import path from "path"
+import url from "url"
 
 const router = Router()
 const streamResponses: Set<Response> = new Set
 
+router.use(urlencoded({ extended: false }))
 router.use(multer().none())
 
 router.post("/redirect", (request, response) => {
-  const path = request.body.path ?? "/src/tests/fixtures/one.html"
-  response.redirect(303, path)
+  const pathname = request.body.path ?? request.query.path ?? "/src/tests/fixtures/one.html"
```

Comment:
@sstephenson I'm reaching the limits of my Express middleware expertise. I was under the impression that the built-in `url encoded` middleware would access values through `request.body`, but it seems like that's most often blank and they're only accessible through `request.query`.

Additionally, any form that specifies `multipart/form-data` errors in a way that I've had trouble debugging, in spite of the `multer().none()` middleware.